skel/manifests/example-pod.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: alpine
+  name: example-pod
 spec:
   restartPolicy: Never
   containers:
-  - name: alpine
+  - name: example
   image: "alpine:3.2"
```

Comment:
One question is should the skeleton chart actually "work"? Debated about setting this to `scratch` and a random command.